app/models/admin/user.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# This user is pulled from Supplejack API
+# For displaying within the admin section of the Manager
+
+module Admin
+  # app/models/admin/user.rb
+  class User
+    attr_accessor :environment, :page
+
+    def initialize(environment, page = 1)
+      @environment = environment
+      @page        = page
+    end
+
+    def all
+      @users = JSON.parse(RestClient.get("#{host}/harvester/users.json", params: { api_key: api_key, page: page }))['users']
+    end
+
```

Comment:
I like this more than active resource because I can understand it, but is there any particular reason we aren't using it?

Reply:
ooo fancy

Reply:
Yeah it's pretty frustrating but once ActiveResource is running you can't change the host, so we can't do the pulling from multiple environments.